addons/website/models/website_page.py

```diff
@@ -170,7 +182,11 @@ def write(self, vals):
             if 'visibility' in vals:
                 if vals['visibility'] != 'restricted_group':
                     vals['group_ids'] = False
+
         self.env.registry.clear_cache()  # write on page == write on view that invalid cache
+        if 'url' in vals or 'visibility' in vals or 'group_ids' in vals:
+            self.env.registry.clear_cache('templates')
```

Comment:
Can you explain the difference between the two clear_cache calls with a comment? :pray: